src/decaylanguage/modeling/ampgen2goofit.py

```diff
@@ -31,7 +31,7 @@ def ampgen2goofit(filename, ret_output=False):
         output = StringIO()
         printer = partial(print, file=output)
     else:
-        printer = print
+        printer = print  # type: ignore
```

Comment:
Proper fix is to predeclare this, but that's Python 3.6 syntax. The other fix would be to add a type on both statements. Or just ignore it.